src/renderer/main/preload.ts

```diff
@@ -57,6 +57,20 @@ window.addEventListener('DOMContentLoaded', async () => {
   ) as HTMLSelectElement;
   setting.setZoomFactor(zoomFactorSelect);
 
+  const doAutoUpdate = store.get('autoUpdate');
+  const autoUpdateRadios = document.getElementsByName('auto-update');
+  autoUpdateRadios.forEach((element: HTMLInputElement) => {
+    if (element.value === doAutoUpdate) {
+      element.checked = true;
+    }
+  });
+  if (!(await ipcRenderer.invoke('is-exe-version'))) {
+    const e = document.getElementById(
+      'auto-update-download'
+    ) as HTMLInputElement;
```

Comment:
一応HTMLInputElementであることを確認するほうが安全には倒せるかもしれないですね・・・？

Reply:
```suggestion
    const e = document.getElementById(
      'auto-update-download'
    );
```

4a7777279b97f9365ee106c7b5b6ec1c93d56085 makes you to allow removing `as`.